### PR TITLE
Light reading: Setup Keybase.io, GPG & Git to sign commits on GitHub

### PR DESCRIPTION
**Pull request by jnareb**
Step-by-step guide on how to create a GPG key on keybase.io, adding it to a local GPG setup and use it with Git and GitHub.

_I am not sure if this link was not published in earlier Git Rev News_

### CODE REVIEW

File: rev_news/drafts/edition-18.md [hunk 1]
```diff
@@ -221,6 +221,7 @@ __Light reading__
 * Max Heiber explains [How I Messed Up Our Repo and Ideas for Improving Git UX](https://medium.com/@maxheiber/how-i-messed-up-our-repo-and-ideas-for-improving-git-ux-e248d9e27e79#.hb10x5w10)
 * [Demystifying Git internals](https://medium.com/@pawan_rawal/demystifying-git-internals-a004f0425a70#.l34w4zy60), by Pawan Rawal
 * [Version control: Effective use, issues and thoughts, from a gamedev perspective](http://www.what-could-possibly-go-wrong.com/version-control/), by Ashley Davis
+* [Setup Keybase.io, GPG & Git to sign commits on GitHub] (and for Git), a gist by Patrick Stadler
```
Comment (moy) [inline rev_news/drafts/edition-18.md]: Isn't this missing a real link? (the `(...)` part after `[...]`)

**Comment by jnareb** [comment]
Yes it is, https://github.com/pstadler/keybase-gpg-github

**Comment by chriscool** [comment]
Indeed it was published in edition 14: https://git.github.io/rev_news/2016/04/20/edition-14/

**Comment by chriscool** [comment]
Anyway, thanks for your pull requests Jakub!

**Comment by chriscool** [comment]
By the way I am not against reposting it in edition 19, if there is little bit more than just the link, like a few more sentences or a full article.

**Pull request closed without merging**